js/canvasmanager.ts

```diff
@@ -3,9 +3,10 @@ import { Pen } from "./drawobjects/pen";
 import { Circle } from "./drawobjects/circle";
 
 import { Point, LineMode, DrawObject } from "./drawobjects/drawobject";
+import { ChangeType } from "./editor";
```

Comment:
The canvas shouldn't depend on the editor. If the canvas manager really needs `ChangeType` (see my other review comment), then `ChangeType` should be moved to `utils.ts`.